jekyll/_cci2/sso/sign-in-to-an-sso-enabled-organization.adoc

```diff
@@ -0,0 +1,32 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Sign in to an SSO-enabled organization
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions outlining how to sign in as an SSO user
+:icons: font
+:experimental:
+
+NOTE: SAML Single Sign-On for CircleCI is currently in closed preview.
+
+[join-an-organization]
+== Join an organization
+
+To join a CircleCI organizaiton that has SSO enabled you must first have an active CircleCI account that you log into with an email and password. For more information on creating a CircleCI account, see the xref:first-steps#[Sign up and try] page.
+
+Once you have a CircleCI account you can join an SSO-enabled CircleCI organization in one of the following ways:
+
+* Get link:https://circleci.com/docs/manage-roles-and-permissions/#add-people-to-your-organization[an invite]
+* Visit the CircleCI link:https://circleci.com/authentication/login?f=sso[SSO login page].
+
+If you have a GitHub or Bitbucket Cloud social login account with CircleCI (that is, you sign in to the CircleCI app using your GitHub/Bitbucket credentials) and you are invited to an SSO-enabled CircleCI organization, you must create a new email/password CircleCI account in order to access the SSO-enabled CircleCI organization.
+
+[log-in-to-an-organization]
+== Log in to an organization
+
+If you have logged in to an SSO-enabled CircleCI organization previously and your session has expired, the next time you attempt to access your organization's resources, you will be redirected to your org's Identity Provider (IdP) to SSO before being granted access once again.
```

Comment:
"to SSO"

Would it be ok to change this to "to authenticate"? or "to authorize"? Or something else

Reply:
Yes, changing it to `to authenticate` sounds good